peppy/project.py

```diff
@@ -356,19 +347,27 @@ def _get_table_from_samples(self, index, initial=False):
         df.set_index(keys=index, drop=False, inplace=True)
         return df
 
-    def parse_config_file(self, cfg_path, amendments=None):
+    def parse_config_file(
+        self,
+        cfg_path: str = None,
+        amendments: Iterable[str] = None,
+        config: dict = None,
```

Comment:
This should be a separate function, I think.

The function signature like this is not clear. This is similar to what we've discussed before -- this seems like there should be two functions, with different signatures, one that calls the other

Reply:
Thanks for mentioning it! I think, I am even not using this argument! I have to check it again manually

Reply:
What are you going to do about this?